apim/3.x/templates/api/api-configmap.yaml

```diff
@@ -116,6 +116,7 @@ data:
     analytics:
       type: elasticsearch
       elasticsearch:
+        enabled: {{ .Values.es.enabled }}
```

Comment:
Apparently this can't be disabled in api config ?

Reply:
Hi @ydbondt 

For now, analytics can't be disable for mgmt-api.
You don't want analytics ?

Reply:
It is not that we don't want analytics, but when we do a proof of concept we don't want to be forced to install a rather big component like elasticsearch.